packages/flutter_tools/lib/src/flutter_manifest.dart

```diff
@@ -151,13 +152,27 @@ class FontAsset {
   String toString() => '$runtimeType(asset: ${assetUri.path}, weight; $weight, style: $style)';
 }
 
-Future<bool> _validate(Object manifest) async {
-  final String schemaPath = fs.path.join(
+@visibleForTesting
+String buildSchemaDir(FileSystem fs){
+  return fs.path.join(
     fs.path.absolute(Cache.flutterRoot), 'packages', 'flutter_tools', 'schema',
+  );
+}
+
+@visibleForTesting
+String buildSchemaPath(FileSystem fs){
+  return fs.path.join(
+    fs.path.absolute(buildSchemaDir(fs)),
```

Comment:
`buildSchemaDir` already returns an absolute path, so this `fs.path.absolute()` shouldn't be needed